### PR TITLE
hw jpa, all tables join

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**

### CODE REVIEW

File: src/main/java/lesson_jpa/ShowTablesInfo.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+	static {
+		Locale.setDefault(Locale.ENGLISH);
+		factory = Persistence.createEntityManagerFactory("PERSISTENCE");
+	}
+
+	private static final Logger LOG = LogManager.getLogger(OrderDaoImpl.class);
+	private static EntityManager entityManager = factory.createEntityManager();
+
+	public static void main(String[] args) {
```
Comment (Daniluko) [inline src/main/java/lesson_jpa/ShowTablesInfo.java:24]: you must create new class

Comment (u2107680007911543774e7r) [reply src/main/java/lesson_jpa/ShowTablesInfo.java:24]: are you sure?)

**Review by Daniluko — APPROVED** [review]

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read message in slack

File: logs/example-logger.log [hunk 2]
```diff
@@ -1133,3 +1133,5 @@ Caused by: java.sql.SQLSyntaxErrorException: ORA-00904: "PRODUCT_ID": invalid id
 | 2019-02-26 21:15:35 | DEBUG | at lesson_jpa.OrderDaoImpl(OrderDaoImpl.java:60) | merge successful | 
 | 2019-02-26 21:15:35 | DEBUG | at lesson_jpa.OrderDaoImpl(OrderDaoImpl.java:74) | removing Order instance | 
 | 2019-02-26 21:15:35 | DEBUG | at lesson_jpa.OrderDaoImpl(OrderDaoImpl.java:79) | remove successful | 
+| 2019-02-28 11:38:42 | DEBUG | at logging.LoggerProviders(LoggerProviders.java:158) | Logging Provider: org.jboss.logging.Log4j2LoggerProvider |
```
Comment (SerhiiPasko) [inline logs/example-logger.log:1136]: ?

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**